src/components/ErrorBlock/Error.tsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+
+interface Props {
+  onClose: () => void;
+}
+
+export const Error: React.FC<Props> = ({ onClose }) => (
+  <div className="notification is-danger is-light has-text-weight-normal">
+    <button
+      type="button"
+      className="delete"
+      onClick={onClose}
+    />
+
+    {/* show only one message at a time */}
+    Unable to add a todo
```

Comment:
This component shows all error messages at once. Consider using a prop to pass the error message to be displayed.

Reply:
this functional i must do at second part, so now it works incorrectly